vue-frontend/src/pages/jobs/apply/[id].vue

```diff
@@ -568,7 +568,7 @@ export default {
                   setTimeout(() => {
                     this.showSuccessMessage = false;
```

Comment:
Will we show a success message after submitting a job application and then redirect? Is it the right flow?
Or we should redirect directly after submission?